config/software/libarchive.rb

```diff
@@ -28,20 +28,19 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  command "./configure" \
-          " --prefix=#{install_dir}/embedded" \
-          " --without-lzma" \
-          " --without-lzo2" \
-          " --without-nettle" \
-          " --without-xml2" \
-          " --without-expat" \
-          " --without-bz2lib" \
-          " --without-iconv" \
-          " --without-zlib" \
-          " --disable-bsdtar" \
-          " --disable-bsdcpio" \
-          " --without-lzmadec" \
-          " --without-openssl", env: env
+  configure("--prefix=#{install_dir}/embedded",
+            "--without-lzma",
+            "--without-lzo2",
+            "--without-nettle",
+            "--without-xml2",
+            "--without-expat",
+            "--without-bz2lib",
+            "--without-iconv",
+            "--without-zlib",
+            "--disable-bsdtar",
```

Comment:
You might actually want to build with bsdtar, xml2, iconv, liblzma, bz2lib and expat on windows.  You could be a thing to investigate later though.